slices/xz-utils.yaml

```diff
@@ -0,0 +1,36 @@
+package: xz-utils
+
+essential:
+  - xz-utils_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - liblzma5_libs
+    contents:
+      /usr/bin/lzmainfo:
+      /usr/bin/unxz:
+      /usr/bin/xz:
+      /usr/bin/xzcat:
+  utils:
+    essential:
+      - base-files_bin
```

Comment:
while I understand this, I don't think this is neccesarily needed, most of shell scripts use `#/bin/sh` which won't be solved by this anyway without any manual setup

Reply:
It is possible to install a shell without the link from  /usr/bin -> /bin. This slice only contains a few directories or links so I think it is ok to add for convenience.